terraform/app/terraform.tf

```diff
@@ -46,14 +46,15 @@ terraform {
   }
 }
 
+
```

Comment:
can we change terraform version in this file to `required_version = ">= 0.12.29"`
to keep it consistent with the pipeline and other files.

Reply:
Pinned in the root (`terraform.tf` and `versions.tf`), but only requiring `>= 0.12` in the modules